### DISCUSSION TITLE
Ability to search for an auth user using phone number on the supabase console/studio

### DISCUSSION THREAD

**Discussion by superakabo** [supabase/supabase#13771] [Feature Requests]
# Feature request
It will be really great to search for an authenticated user using phone number on the Supabase console.

## Is your feature request related to a problem? Please describe.
At the moment one can only search for an authenticated user from the authentication page using email address.

A clear and concise description of what you want and what your use case is.
I want to search or filter the list of authenticated users on the supabase console using phone number.

## Describe the solution you'd like
I want to search for an authenticated user using either phone number or email address on the Authentication table of the Supabase console.

When I go to the Supabase Studio, and I type a phone number or email address it should filter the list of authenticated users and return matching records.

## Describe alternatives you've considered

https://github.com/supabase/gotrue-js/issues/359
Accessing the Postgres database from an IDE such as intelliJ.

## Additional context

<img width="1673" alt="Screenshot 2022-12-03 at 11 57 41 AM" src="https://user-images.githubusercontent.com/11073056/205439803-a2287536-4dc8-403a-9303-3f3690100eb2.png">

**Comment by hf** [comment]
Added to our internal feedback board.
